.circleci/config.yml

```diff
@@ -62,10 +62,21 @@ jobs:
             docker push "quay.io/mojanalytics/control-panel:${CIRCLE_BRANCH}"
 
       - deploy:
-          name: Deploy to Kubernetes
+          name: Deploy to dev
           command: |
             curl ${JENKINS_URL}/job/control-panel/job/api/buildWithParameters \
               --data token=${JENKINS_JOB_TRIGGER_TOKEN} \
               --data COMMIT_HASH=${CIRCLE_SHA1} \
               --data BRANCH_NAME=${CIRCLE_BRANCH} \
               --user ${JENKINS_USER}:${JENKINS_TOKEN}
+
+      - deploy:
+          name: Deploy to alpha
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "api" ]; then
```

Comment:
I was about to ask if there was a `branch:` field to use instead...

But looking at CircleCI documentation looks like that's how they're doing it there.